app/boards/arm/glove80/glove80_rh.dts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+
+#include "glove80.dtsi"
+#include "glove80_rh-pinctrl.dtsi"
+
+#include <dt-bindings/led/led.h>
+
+/ {
+    model = "glove80_rh";
+    compatible = "glove80_rh";
+
+    chosen {
+        zmk,underglow = &led_strip;
+        zmk,backlight = &back_led_backlight;
+        zmk,battery   = &vbatt;
+    };
+
+    back_led_backlight: pwmleds {
+        compatible = "pwm-leds";
+        label = "BACK LED";
+        pwm_led_0 {
+            pwms = <&pwm0 0 PWM_USEC(20) PWM_POLARITY_NORMAL>;
+            label = "Back LED configured as backlight";
+        };
+    };
+
+    ext-power {
+        compatible = "zmk,ext-power-generic";
+        label = "EXT_POWER";
+        control-gpios = <&gpio0 19 GPIO_ACTIVE_HIGH>; /* WS2812_CE */
+        init-delay-ms = <100>;
+    };
+
+    vbatt: vbatt-divider {
+        compatible = "zmk,battery-voltage-divider";
+        label = "BATTERY";
+        io-channels = <&adc 5>;
+        output-ohms = <820000>;
+        full-ohms = <(1500000 + 820000)>;
+    };
+
+    vbatt-vddh {
+        compatible = "zmk,battery-nrf-vddh";
+        label = "BATTERY-VDDH";
+    };
+};
+
+&spi1 {
```

Comment:
This should be moved to SPI3, see #1746 for the details and why-for.

Reply:
Thanks, I'll make that change.